starlite/asgi/routing_trie/traversal.py

```diff
@@ -82,6 +85,9 @@ def parse_path_params(
         parameter_definitions: The parameter definitions tuple from the route.
         path_param_values: The string values extracted from the url
 
+    Raises:
+        ValueError: If any of path parameters can not be parsed into a value.
```

Comment:
While `parse_path_params` doesn't explicitly raises `ValueError` it is raised indirectly so I decided to update the docsstring here b/c it took me some time to understand sources of `KeyError` and `ValueError`  exceptions catched in `parse_path_to_route`